### PR TITLE
[Schema Registry] Make display name consistent across languages

### PR DESCRIPTION
**Pull request by deyaaeldeen**
I decided to drop the word `Serializer` from the display name for the avro packages to make names shorter but I can do it the other way if you're interested.

### CODE REVIEW

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -20,7 +20,7 @@
 "azure-eventhub-checkpointstoreblob","1.1.4","","Event Hubs - Azure Blob Storage Checkpoint Store","Event Hubs","eventhub","","","client","true","","01/14/2020","active","","",""
 "azure-eventhub-checkpointstoreblob-aio","1.1.4","","Event Hubs - Azure Blob Storage Checkpoint Store AIO","Event Hubs","eventhub","","","client","true","","01/14/2020","active","","",""
 "azure-schemaregistry","","1.0.0b2","Event Hubs - Schema Registry","Event Hubs","schemaregistry","","","client","true","","","preview","","",""
-"azure-schemaregistry-avroserializer","","1.0.0b2","Event Hubs - Schema Registry Avro Serializer","Event Hubs","schemaregistry","","","client","true","","","preview","","",""
+"azure-schemaregistry-avroserializer","","1.0.0b2","Event Hubs - Schema Registry Avro","Event Hubs","schemaregistry","","","client","true","","","preview","","",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:23]: Should the service name be "Event Hubs" or "Schema Registry"? Also please check the other languages if you haven't already.

Comment (deyaaeldeen) [reply _data/releases/latest/python-packages.csv]: I updated it to Schema Registry.

File: _data/releases/latest/java-packages.csv [hunk 2]
```diff
@@ -512,7 +512,7 @@
 "azure-messaging-eventhubs-parent","com.azure","","5.0.0-preview.1","Event Hubs - Parent","Event Hubs","NA","NA","NA","client","false","","","","","",""
 "azure-eventhubs-reactive_2.12","com.microsoft.azure","","0.5.0","Event Hubs - Reactive","Event Hubs","NA","NA","NA","client","false","","","","","",""
 "azure-event-hubs-reactive_2.12","com.microsoft.azure","","0.1.0","Event Hubs - Reactive","Event Hubs","NA","NA","NA","client","false","","","","true","",""
-"azure-schemaregistry-kafka-avro","com.microsoft.azure","","1.0.0-beta.4","Event Hubs - Schema Registry Kafka Avro","Event Hubs","NA","NA","NA","client","false","","","","","",""
+"azure-schemaregistry-kafka-avro","com.microsoft.azure","","1.0.0-beta.4","Schema Registry Kafka Avro","Schema Registry","NA","NA","NA","client","false","","","","","",""
```
Comment (JoshLove-msft) [inline _data/releases/latest/java-packages.csv:515]: Why does this one have Kafka?

File: _data/releases/latest/dotnet-packages.csv [hunk 3]
```diff
@@ -45,7 +45,7 @@
 "Azure.Analytics.Purview.Catalog","","1.0.0-beta.1","Purview Catalog","Purview","purview","","","client","true","","","","","",""
 "Azure.Analytics.Purview.Scanning","","1.0.0-beta.1","Purview Scanning","Purview","purview","","","client","true","","","","","",""
 "Azure.AI.Language.QuestionAnswering","","1.0.0-beta.1","Question Answering","Cognitive Language","cognitivelanguage","","","client","true","","","","","",""
-"Microsoft.Azure.Data.SchemaRegistry.ApacheAvro","","1.0.0-beta.2","Event Hubs - Schema Registry Avro","Schema Registry","schemaregistry","NA","","client","true","1.0.0-beta.3,10/05/2021","","","","",""
+"Microsoft.Azure.Data.SchemaRegistry.ApacheAvro","","1.0.0-beta.2","Schema Registry Avro","Schema Registry","schemaregistry","NA","","client","true","1.0.0-beta.3,10/05/2021","","","","",""
```
Comment (JoshLove-msft) [inline _data/releases/latest/dotnet-packages.csv:48]:
```suggestion
"Microsoft.Azure.Data.SchemaRegistry.ApacheAvro","","1.0.0-beta.2","Schema Registry - Avro","Schema Registry","schemaregistry","NA","","client","true","1.0.0-beta.3,10/05/2021","","","","",""
```

File: _data/releases/latest/java-packages.csv (hunk 2 above)
Comment (deyaaeldeen) [reply _data/releases/latest/java-packages.csv:515]: There is a separate one for avro at L36. @conniey do you know anything about this package? could it be https://github.com/Azure/azure-schema-registry-for-kafka? /cc @ramya-rao-a

File: _data/releases/latest/dotnet-packages.csv (hunk 3 above)
Comment (deyaaeldeen) [reply _data/releases/latest/dotnet-packages.csv:48]: Is it just a matter of preference or is there another reason to do this change? note that it has to be done for all languages.

Comment (JoshLove-msft) [reply _data/releases/latest/dotnet-packages.csv:48]: Just a preference, but I thought it reads better, especially if we end up with multiple serializer packages, e.g. Schema Registry - Avro, Schema Registry - Protobuf, etc.

Comment (deyaaeldeen) [reply _data/releases/latest/dotnet-packages.csv]: Done!

**Review by JoshLove-msft — APPROVED** [review]

File: _data/releases/latest/java-packages.csv (hunk 2 above)
Comment (ramya-rao-a) [reply _data/releases/latest/java-packages.csv]: We shouldnt have the kafka one here, we don't own that package

File: _data/releases/latest/java-packages.csv [hunk 4]
```diff
@@ -512,7 +512,7 @@
 "azure-messaging-eventhubs-parent","com.azure","","5.0.0-preview.1","Event Hubs - Parent","Event Hubs","NA","NA","NA","client","false","","","","","",""
 "azure-eventhubs-reactive_2.12","com.microsoft.azure","","0.5.0","Event Hubs - Reactive","Event Hubs","NA","NA","NA","client","false","","","","","",""
 "azure-event-hubs-reactive_2.12","com.microsoft.azure","","0.1.0","Event Hubs - Reactive","Event Hubs","NA","NA","NA","client","false","","","","true","",""
-"azure-schemaregistry-kafka-avro","com.microsoft.azure","","1.0.0-beta.4","Event Hubs - Schema Registry Kafka Avro","Event Hubs","NA","NA","NA","client","false","","","","","",""
+"azure-schemaregistry-kafka-avro","com.microsoft.azure","","1.0.0-beta.4","Schema Registry Kafka - Avro","Schema Registry","NA","NA","NA","client","false","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:515]:
```suggestion
"azure-schemaregistry-kafka-avro","com.microsoft.azure","","1.0.0-beta.4","Schema Registry - Kafka Avro","Schema Registry","NA","NA","NA","client","false","","","","","",""
```

**Review by weshaggard — APPROVED** [review]

File: _data/releases/latest/java-packages.csv (hunk 2 above)
Comment (deyaaeldeen) [reply _data/releases/latest/java-packages.csv]: I opened https://github.com/Azure/azure-sdk/pull/3393

Comment (conniey) [reply _data/releases/latest/java-packages.csv]: I have no idea what package that is.

**Pull request merged**